src/main/java/org/skriptlang/skript/bukkit/spark/expressions/ExprMSPT.java

```diff
@@ -0,0 +1,75 @@
+package org.skriptlang.skript.bukkit.spark.expressions;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.RequiredPlugins;
+import org.skriptlang.skript.bukkit.spark.SparkUtils;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.ExpressionType;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.lang.util.SimpleExpression;
+import ch.njol.util.Kleenean;
+import me.lucko.spark.api.Spark;
+import me.lucko.spark.api.SparkProvider;
+import me.lucko.spark.api.statistic.StatisticWindow;
+import me.lucko.spark.api.statistic.misc.DoubleAverageInfo;
+import me.lucko.spark.api.statistic.types.GenericStatistic;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+
+@Name("MSPT Usage")
+@RequiredPlugins("Paper 1.21+ or Spark")
+@Description({
+    "Returns the MSPT (milliseconds per tick) readings from the last 10 seconds, 1 minute and 5 minutes.",
+    "This expression can only be used if the server has Spark or you have Spark installed as a plugin."
+})
+@Examples({
+	"broadcast server tick"
+})
+public class ExprMSPT extends SimpleExpression<Number> {
+
+	private int index;
+	private String expr = "server tick";
+	private static Spark spark;
+
+	static {
+		if (Skript.classExists("me.lucko.spark.api.SparkProvider")) {
+			Skript.registerExpression(ExprMSPT.class, Number.class, ExpressionType.SIMPLE,
+				"[the] (server tick (duration|time)|mspt) (from|of) the last 10 seconds",
+				"[the] (server tick (duration|time)|mspt) (from|of) the last [(1|one) ]minute",
+				"[the] (server tick (duration|time)|mspt) (from|of) the last (5|five) minutes",
```

Comment:
I think `the` should be an optional `[the]` here. `server tick time from last 10 seconds` makes perfect sense.